scipy/stats/tests/test_distributions.py

```diff
@@ -1197,14 +1197,6 @@ def test_cdf_tail_15110_14753(self):
         assert_allclose(stats.truncnorm(8, np.inf).cdf(8.3),
                         0.9163220907327540)
 
-    def _test_moments_one_range(self, a, b, expected, rtol=1e-7):
```

Comment:
As far as I can tell this is equivalent to https://github.com/scipy/scipy/blob/5955e906c56fa5f7ec9837f56e44a04a7a4d438f/scipy/stats/tests/test_distributions.py#L1242-L1245